cmd/ris-mirror/rismirror/router.go

```diff
@@ -12,10 +13,12 @@ import (
 
 // Router represents a router
 type Router struct {
-	name        string
-	address     net.IP
-	vrfRegistry *vrf.VRFRegistry
-	vrfs        map[uint64]*_vrf
+	name         string
+	address      net.IP
+	vrfRegistry  *vrf.VRFRegistry
+	vrfs         map[uint64]*_vrf
```

Comment:
I think access to `vrfs` also needs to be synchronized or am I missing something?

Reply:
Moved the mutex to vrf access, as the clients are now inside the vrfs.